### PR TITLE
Text fixes.

### PR DESCRIPTION
**Pull request by Lorantine**

### CODE REVIEW

File: data/story northern union.txt [hunk 1]
```diff
@@ -392,72 +392,71 @@ mission "Meeting an Alpha 1"
...
 				`	"Sorry, but I can't make money when I'm dead."`
 					decline
-				`	"Scan a battle cruiser? Do you think I'm suicidal?"`
+				`	"Scan a battlecruiser? Do you think I'm suicidal?"`
 					decline
 	
 	on accept
 		log "Volunteered to scan the Star Blade, one of the largest vessels in human space."
-		dialog "Upon returning to your ship, you check your communicator, only to find that you have two messages. Upon opening the second one, you find out this belonged to Peg and is full of intel that could be useful to the Republic. Maybe they'll pay you for it?"
+		dialog "Upon returning to your ship, you check your communicator and find that you have received two messages. Upon opening the second one, you find out it belonged to Peg and is full of intel that could be useful to the Republic. Maybe they'll pay you for it?"
```
Comment (kikotheexile) [inline data/story northern union.txt:441]: "messages" was meant to be "communicators" 

Two phones essentially. I messed that up.

**Comment by kikotheexile** [comment]
The one big error I spotted was mine. The writing could be sanded down a bit more so its not as sharp/pointed as mine, but all told the spelling/grammar fixes look fine.

**Comment by Lorantine** [comment]
I deliberately stick to fixing what's there, not improving dialogue as that'll take way longer and I'm less proficient in it.

Improving dialogue is for others, later. However, I hope to have those writing guidelines/tips on Discord within a week, or at least an initial form of it.

TODO (for myself):
Fix messages > communicators and restructure the sentence for it.

Fix will be included in my PR with Navy and/or Alpha story text fixes, similar to this one.

**Comment by kikotheexile** [comment]
thats understandable, and thank you

**Pull request merged**